src/plugins/GUITestBase/src/GTUtilsOptionPanelSequenceView.cpp

```diff
@@ -360,6 +362,42 @@ void GTUtilsOptionPanelSequenceView::openAnnotationParametersShowHideWidget(HI::
 }
 #undef GT_METHOD_NAME
 
+#define GT_METHOD_NAME "openInSilicoPcrMeltingTempertureShowHideWidget"
+void GTUtilsOptionPanelSequenceView::openInSilicoPcrMeltingTempertureShowHideWidget(HI::GUITestOpStatus& os) {
+    GTWidget::click(os, GTWidget::findWidget(os, "ArrowHeader_Melting temperature"));
+}
+#undef GT_METHOD_NAME
+
+#define GT_METHOD_NAME "showMeltingTemperatureDialog"
+void GTUtilsOptionPanelSequenceView::showMeltingTemperatureDialog(HI::GUITestOpStatus& os) {
+    GTUtilsOptionPanelSequenceView::openTab(os, GTUtilsOptionPanelSequenceView::Statistics);
+    GTUtilsOptionsPanel::resizeToMaximum(os);
+
+    auto statisticsLabel = GTWidget::findLabel(os, "Common Statistics");
+
+    QPoint labelGlobalPos = statisticsLabel->mapToGlobal(statisticsLabel->pos());
+    GTMouseDriver::moveTo(labelGlobalPos);
+    for (int column = 0; column < statisticsLabel->height(); column += 10) {
+        bool found = false;
+        for (int row = 0; row < statisticsLabel->width(); row += 10) {
+            GTThread::waitForMainThread();
+            QPoint newMousePos = labelGlobalPos + QPoint(column, row);
+            GTMouseDriver::moveTo(newMousePos);
+            Qt::CursorShape shape = statisticsLabel->cursor().shape();
+            if (shape != Qt::ArrowCursor) {
+                GTMouseDriver::click();
+                found = true;
```

Comment:
no need to use `bool found`: just return

Reply:
OK